qube/create/templates/portlet/portlet_groovy_osgi/{{ cookiecutter.project_slug }}/README.rst

```diff
@@ -0,0 +1,40 @@
+{{ cookiecutter.project_name }}
+-----------------------------------
+
+.. image:: https://github.com/qbicsoftware/{{ cookiecutter.project_slug }}/workflows/Build%20Maven%20Package/badge.svg
+    :target: https://github.com/qbicsoftware/{{ cookiecutter.project_slug }}/workflows/Build%20Maven%20Package/badge.svg
+    :alt: Github Workflow Build Maven Package Status
+
+.. image:: https://github.com/qbicsoftware/{{ cookiecutter.project_slug }}/workflows/Run%20Maven%20Tests/badge.svg
+    :target: https://github.com/qbicsoftware/{{ cookiecutter.project_slug }}/workflows/Run%20Maven%20Tests/badge.svg
+    :alt: Github Workflow Tests Status
+
+.. image:: https://github.com/qbicsoftware/{{ cookiecutter.project_slug }}/workflows/QUBE%20lint/badge.svg
+    :target: https://github.com/qbicsoftware/{{ cookiecutter.project_slug }}/workflows/QUBE%20lint/badge.svg
+    :alt: qube Lint Status
+
+.. image:: https://readthedocs.org/projects/{{ cookiecutter.project_slug | replace("_", "-") }}/badge/?version=latest
+    :target: https://{{ cookiecutter.project_slug | replace("_", "-") }}.readthedocs.io/en/latest/?badge=latest
+    :alt: Documentation Status
+
+.. image:: https://flat.badgen.net/dependabot/thepracticaldev/dev.to?icon=dependabot
+    :target: https://flat.badgen.net/dependabot/thepracticaldev/dev.to?icon=dependabot
+    :alt: Dependabot Enabled
+
+
+{{ cookiecutter.project_short_description }}
+
+* Free software: {{ cookiecutter.license }}
```

Comment:
Here is where I referenced the license. Not sure if it is important for the bug that was introduced with adding template.